posts/workshop-vignette/index.qmd

```diff
@@ -0,0 +1,349 @@
+---
+title: "workshop vignette"
+output: 
+  pdf_document:
+    latex_engine: xelatex
+date: "2023-01-12"
```

Comment:
Could you add yourself as author and add some categories (i.e., tags or keywords) please?

Docs: https://quarto.org/docs/websites/website-blog.html#categories